restaurant_bill/src/tests/calculate.test.js

```diff
@@ -0,0 +1,20 @@
+import calculateBill from '../scripts/calculate'
+
+
+test('calculates the exact value to pay', () => {
+  const people = 2;
+  const totalBill = 10;
+  const extraTip = 0;
+  const result = 5;
+  const bill = calculateBill({ people, totalBill, extraTip })
+  expect(bill).toBe(result);
+});
+
+test('calculates the exact value to pay', () => {
+  const people = 2;
+  const totalBill = 9;
+  const extraTip = 0;
+  const result = 4.5;
+  const bill = calculateBill({ people, totalBill, extraTip })
+  expect(bill).toBe(result);
+});
```

Comment:
Love the fact that you've written real tests!

Reply:
I use to do this when I am creating a function to do something specific like this calculation, Its a custum I have, even I write test while solving coding challenges, it really saves you time and effort [my coding challenges tests](https://github.com/carloshdelreal/coding_challenges)